Library/ViewModels/BackerDashboardViewModelTests.swift

```diff
@@ -221,4 +251,14 @@ internal final class BackerDashboardViewModelTests: TestCase {
 
     XCTAssertEqual(-500, self.vm.outputs.initialTopConstant)
   }
+
+  /*
+   Helper for testing discoverProperties Watched Filter
+   */
+
+  private func assertDiscoveryProperties(_ props: [String: Any]?) {
```

Comment:
I see why you added this but also notice we don't do this outside of the KSRAnalyticsTests file. Probably not a big deal if we delete this here because it's being tested in another file?